### DISCUSSION TITLE
Make the Timeline sub menu easier to find.

### DISCUSSION THREAD

**Discussion by DavidBailes** [audacity/audacity#5931] [Development]
Currently keyboard users can change the view of the timeline using the Timeline sub menu in the Extra menu.
In Audacity 3.5 this setting affects what happens to the import of music files - see the table in: https://github.com/audacity/audacity/pull/5570
So given that this will be an important setting in 3.5, I think it would be better if this sub menu was easier to find, rather than being in a menu which is not shown by default. An obvious place would be the View menu. Indeed, in the Import/Export page of preferences, there is the text "switch view to beats and measures".

**Comment by LWinterberg** [comment]
The Extra menu really is a crutch I wish we could get rid of. But of course, as long as so much of the UI is inaccessible, it's not an option.

About your question: I don't mind it being moved to the view menu for exposure, into a section with the Toolbars item.
